src/Tribe/Ticket_Object.php

```diff
@@ -660,5 +698,16 @@ public function get_event() {
 
 			return null;
 		}
+
+		/**
+		 *
+		 */
+		public function show_description() {
+				$show = ( metadata_exists( 'post', $this->ID, $this->show_description_key ) ) ?
```

Comment:
If the ternary is too long to be readable on one line, might as well make it readable by switching to an `if` statement.

```php
$show = true;

if ( metadata_exists( 'post', $this->ID, $this->show_description_key ) ) {
  $show = get_post_meta( $this->ID, $this->show_description_key, true );
}
```

If you really like the ternary, then formatting like this would be preferred:

```php
$show = metadata_exists( 'post', $this->ID, $this->show_description_key )
  ? get_post_meta( $this->ID, $this->show_description_key, true )
  : true;
```